docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -71,6 +71,12 @@ Value-specific variables are available under `__value` namespace:
 - `__value.text` - text representation of a value
 - `__value.calc` - calculation name if the value is result of calculation
 
+## Data variables
+
+Data-specific variables are available under `__data` namespace:
+
+- `__data.fields[i]` - value of data field, `i` is an index starting from 0
```

Comment:
```suggestion
To access values and labels from other fields use 

- `${__data.fields[i]}` - value of field i (on the same row)
- `${__data.fields["NameOfField"]}` - value of field using name instead of index
- `${__data.fields["NameOfField"]}` - value of field using name instead of index
- `${__data.fields[1].labels.cluster}` - Access labels of another field 
```